eng/Version.Details.xml

```diff
@@ -7,4 +7,122 @@
+    </Dependency>
+    <Dependency Name="NuGet.Build.Tasks" Version="5.0.0-preview1.5754">
+      <Uri>https://github.com/NuGet/NuGet.Client</Uri>
+      <Sha>7a5c6ae18ae07c22a6e3ad46f18f558a75830955</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview4.19165.1">
+      <Uri>https://github.com/dotnet/sdk</Uri>
+      <Sha>51aba9eaa1b98c313df7a77bc108846cfb7ba51d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Cli.CommandLine" Version="1.0.0-preview.19127.1">
+      <Uri>https://github.com/dotnet/cliCommandLineParser</Uri>
+      <Sha>f011de8f90a23cdef7887a97d5f496570eb1808d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.ApplicationInsights" Version="2.0.0">
+      <Uri>https://github.com/Microsoft/ApplicationInsights-dotnet</Uri>
+      <Sha>53b80940842204f78708a538628288ff5d741a1d</Sha>
+    </Dependency>
+    <Dependency Name="XliffTasks" Version="0.2.0-beta-000048">
+      <Uri>https://github.com/dotnet/xliff-tasks</Uri>
+      <Sha>27d43b762aa6dac3a0a6ba48fe55000942d75c1c</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Cli.Runtime" Version="3.0.100-preview4.19164.1">
+      <Uri>https://github.com/dotnet/cli</Uri>
+      <Sha>1237c39d038231589c0c9270bc85f69e2c82686c</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.FSharp.Compiler" Version="10.4.2-rtm-190302-01">
+      <Uri>https://github.com/microsoft/visualfsharp</Uri>
+      <Sha>b6d8b38f093d98d3e8511d2e999c922db53bac4a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.Web" Version="3.0.100-preview4.19167.1">
+      <Uri>https://github.com/aspnet/websdk</Uri>
+      <Sha>f5c7292b79d5a6cd98e6065558608116b6d0e8d8</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.TestPlatform.Build" Version="15.9.0-preview-20180807-05">
+      <Uri>https://github.com/Microsoft/vstest</Uri>
+      <Sha>df96f832f28de9798bf3a1b2d6b7933439e00ae4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Dotnet.Toolset.Internal" Version="3.0.100-preview4.19167.1">
+      <Uri>https://github.com/dotnet/toolset</Uri>
+      <Sha>b27a1cadc9f5e1d1d347a2fa3706b6203c68ece8</Sha>
+    </Dependency>
+    <Dependency Name="core-sdk" Version="3.0.100-preview3">
+      <Uri>https://github.com/dotnet/core-sdk</Uri>
+      <Sha>d72abce2133c7e37c524f7933bce09a2f44e788f</Sha>
+    </Dependency>
+
+    <!-- fake dependencies, not handled by Maestro/Arcade -->
+    <Dependency Name="FSharp.NET.Sdk" Version="1.0.4-bundled-0100">
+      <Uri>https://github.com/dotnet/source-build/netcorecli-fsc</Uri>
+      <Sha>6f29a34e3e2e429a379c9de1b0e917a524d0be16</Sha>
+    </Dependency>
+
+    <!-- external dependencies, not handled by Maestro/Arcade -->
+    <Dependency Name="Newtonsoft.Json" Version="9.0.1">
+      <Uri>https://github.com/dseefeld/Newtonsoft.Json</Uri>
+      <Sha>e43dae94c26f0c30e9095327a3a9eac87193923d</Sha>
+    </Dependency>
+
+    <!-- not yet implemented -->
```

Comment:
Do we want these in here if they're not yet implemented?

Reply:
I think for ASP.Net stuff it makes sense to keep it in here since that is planned to eventually be source-buildable, but I'll remove the Windows Desktop SDK since we're not planning on supporting it.  Does that sound reasonable?

Reply:
Yes.  That makes sense.